### DISCUSSION TITLE
Github Education doesn't let me upload proof of academic status

### DISCUSSION THREAD

**Discussion by haresh02** [community/community#110802] [GitHub Education] [answered]
### Hi everyone,

Seems like I can't progress the "Upload proof of academic status" page when trying to register for Github Education. I'm trying to upload an image of my offer letter which should fit the requirements but when I click upload it just resets the page. I've also waited a week or so thinking it may just be processing but I don't think that's the case anymore. Thanks for any help!

**Comment by SebastienXia** [comment, marked as answer]
Hi, @haresh02 

I had the same problem, if the download does not work, just do the other option which is to take a photo of your certificate and confirm the application. :+1:

**Reply by polypores** [reply]
It worked. Thanks

**Comment by Duncanmck** [comment]
I have the same issue
